ui/src/actions/__tests__/citations.test.js

```diff
@@ -63,24 +62,15 @@ describe('citations - async action creator', () => {
       { type: types.CITATIONS_SUMMARY_SUCCESS, payload: { foo: 'bar' } },
     ];
 
-    const store = getStoreWithState({
-      router: {
-        location: { pathname: '/authors/12345' },
-      },
-      authors: fromJS({
-        publications: {
-          query: {
-            author: ['J.M.Maxson.1_Jared'],
-          },
-        },
-      }),
-    });
-    await store.dispatch(fetchCitationSummary());
+    const store = getStore();
+    await store.dispatch(
+      fetchCitationSummary({ author: ['J.M.Maxson.1_Jared'] })
```

Comment:
correct my if I'm wrong but we changed the BAI, shall we update these tests (since we're touching them) to reflect the changes? (I know it's not important but just as a guide)